### PR TITLE
fix migration to not include spaces and remove old folder

### LINKED ISSUE #825: Cluster icon doesn't get migrated from 3.5 to 3.6
**Describe the bug**
A cluster icon that was added in the 3.5 version is not used (default icon is shown) after updating to 3.6

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <smalton@mirantis.com>

fixes #825

### CODE REVIEW

File: src/main/logger.ts [hunk 1]
```diff
@@ -1,6 +1,27 @@
 import { app, remote } from "electron";
 import winston from "winston"
 import { isDebugging } from "../common/vars";
+import { format } from "logform";
+import stringify from "string.ify";
+import { MESSAGE } from "triple-beam";
+
+const debugSimple = format(info => {
```
Comment (jakolehm) [inline src/main/logger.ts:8]: Should not be part of this PR.

**Review by jakolehm — CHANGES_REQUESTED** [review]

File: src/main/logger.ts (hunk 1 above)
Comment (Nokel81) [reply src/main/logger.ts:8]: I understand, however I really think that this should be part of 3.6.0 because when `DEBUG=true` is passed the application freezes whenever it prints out the cluster store json object.

File: src/main/logger.ts [hunk 2]
```diff
@@ -1,6 +1,27 @@
 import { app, remote } from "electron";
 import winston from "winston"
 import { isDebugging } from "../common/vars";
+import { format } from "logform";
+import stringify from "string.ify";
+import { MESSAGE } from "triple-beam";
```
Comment (ixrock) [inline src/main/logger.ts:6]: Why so many random packages? Can't we handle this easily with vanilla JS?

File: types/mocks.d.ts [hunk 3]
```diff
@@ -3,6 +3,7 @@ declare module "mac-ca"
 declare module "win-ca"
 declare module "@hapi/call"
 declare module "@hapi/subtext"
+declare module "string.ify"
```
Comment (ixrock) [inline types/mocks.d.ts:6]: I would prefer avoid modules without types nowadays without strong reason to include it.

File: src/main/logger.ts (hunk 2 above)
Comment (jim-docker) [reply src/main/logger.ts:6]: I agree, we need to be more careful about importing third-party modules

Comment (Nokel81) [reply src/main/logger.ts:6]: These are all already dependencies, namely of `winston`. These lines were inspired by the logform package's simple formatter.

**Comment by Nokel81** [comment]
Closing due to theis basically just being a logging format PR change. Will make a ticket for it instead to discuss options.
